src/meta/src/hummock/compaction/compaction_config.rs

```diff
@@ -21,7 +21,7 @@ const DEFAULT_MIN_COMPACTION_BYTES: u64 = 128 * 1024 * 1024; // 128MB
 const DEFAULT_MAX_BYTES_FOR_LEVEL_BASE: u64 = 512 * 1024 * 1024; // 512MB
 
 // decrease this configure when the generation of checkpoint barrier is not frequent.
-const DEFAULT_TIER_COMPACT_TRIGGER_NUMBER: u64 = 6;
+const DEFAULT_TIER_COMPACT_TRIGGER_NUMBER: u64 = 8;
```

Comment:
why we increase the `DEFAULT_TIER_COMPACT_TRIGGER_NUMBER` from 6 to 8 ?

It may affect the scores, is it to lower the priority of l0 intra?